API_INAD/INAD-integration-with-ANPR.yaml

```diff
@@ -32,7 +32,7 @@ paths:
         200:
           description: OK
           content:
-            multipart/form-data:
+            application/json:
```

Comment:
ho dubbi sulla dimensione della risposta... avevo letto che lavorando con multipart/form-data era meglio.... a te risuta che è indifferente application/json

grazie

Reply:
Non credo ci sia differenza, ed inoltre mi sembra insolito avere multipart/form-data  su una response, solitamente si usa sulle request. 

Condivido pero le perplessita sulla dimensione sulla risposta, che andrebbero indirizzate in qualche modo, penso a paginazione o quanto meno uso di range request, sarebbe da approfondire